obal/data/roles/diff_brew_package/defaults/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+diff_brew_pkg_dists:
+  - name: rhel5
```

Comment:
Is this here for example purposes? Obal shouldn't carry specifics

Reply:
Yes, this data is only there to drive the current brew case while the refactoring is happening